metricbeat/module/jolokia/jmx/config.go

```diff
@@ -491,19 +491,17 @@ func (pc *JolokiaHTTPPostFetcher) Fetch(m *MetricSet) ([]common.MapStr, error) {
 		}
 	}
 
-	http, err := helper.NewHTTP(m.BaseMetricSet)
+	m.http.SetMethod(httpReqs[0].HTTPMethod)
+	m.http.SetBody(httpReqs[0].Body)
 
-	http.SetMethod(httpReqs[0].HTTPMethod)
-	http.SetBody(httpReqs[0].Body)
-
-	resBody, err := http.FetchContent()
+	resBody, err := m.http.FetchContent()
```

Comment:
Notice that there is also a `JolokiaHTTPGetFetcher` to use the GET method, could you please make these changes also there?

Reply:
Done, thanks!